package.json

```diff
@@ -84,7 +84,7 @@
     "sls": "LOCAL_PLUGIN=true SLS_DEBUG=* cd example && yarn && npm run build && cd ../",
     "slsDeploy": "LOCAL_PLUGIN=true SLS_DEBUG=* cd example && yarn && npm run deploy && cd ../",
     "test": "npm run eslint && npm run build && npm run jest && npm run sls",
-    "uglify": "uglifyjs dist/index.js --output dist/index.js --beautify",
+    "uglify": "./node_modules/uglify-es/bin/uglifyjs dist/index.js --output dist/index.js --beautify",
```

Comment:
For some reason the `.bin` version of `uglify-es` was pointing towards the `uglify` package, which doesn't handle es6 and is the whole reason for `uglify-es`. I don't know if there was a botched npm deploy on their side or what. Super strange